stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/diff/ChannelListDiffCallback.kt

```diff
@@ -0,0 +1,73 @@
+package io.getstream.chat.android.ui.channel.list.adapter.diff
+
+import androidx.recyclerview.widget.DiffUtil
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.ui.utils.extensions.getLastMessage
+import io.getstream.chat.android.ui.utils.extensions.getLastMessageReadCount
+import io.getstream.chat.android.ui.utils.extensions.getUsers
```

Comment:
Seems code style violation. It's better to check your branch via ktlint (seems we don't do it automatically)

Reply:
Which part is a code style violation? I can't commit via git if I have ktlint errors, so this is all valid for ktlint.

Reply:
It seems like a violation
![image](https://user-images.githubusercontent.com/11807573/99387126-7ffeed80-28e4-11eb-9b2b-fbb25b208d81.png)

Reply:
Ah, I see. I'll fix that.